### PR TITLE
Don't display previous proposals card link unless there are more propo…

### PR DESCRIPTION
**Pull request by HSalmon**
…sals to show

Addresses: https://github.com/loomio/loomio/issues/3334

Not sure we want to be serializing more stuff out but yeah, first go

### CODE REVIEW

File: app/models/group.rb [hunk 1]
```diff
@@ -424,6 +424,10 @@ def organisation_motions_count
     Discussion.published.where(group_id: org_group_ids).sum(:motions_count)
   end
 
+  def closed_motions_count
```
Comment (gdpelican) [inline app/models/group.rb]: Can't do this, unfortunately, it'll be too slow. If we want this feature, we'll have to make a counter cache column for it, `closed_motions_count`, on the group model.

**Comment by HSalmon** [comment]
And what do you reckon? Do we want this feature?

**Comment by HSalmon** [comment]
@robguthrie, what do you reckon? Do we want this feature right now?

**Comment by robguthrie** [comment]
Yeah. Seems like a nice thing to have. A counter cache is pretty easy to add.
We use custom_counter_cache for this kind of thing. It's actually pretty great.

If you add the counter, I'll help you write a migration SQL statement to populate it for existing groups.

**Comment by gdpelican** [comment]
:+1:

**Pull request closed without merging**